lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -271,14 +271,14 @@ function _campaign_resource_retrieve($nid) {
 /**
  * Retrieve and show index list response of campaigns requested.
  *
- * @param string $ids
- * @param bool $staff_pick
- * @param bool $mobile_app
- * @param string $mobile_app_date
- * @param string $term_ids
- * @param int $count
- * @param bool $random
- * @param int $page
+ * @param  string  $ids
```

Comment:
@DFurnes that's part of the beauty of Laravel's approach... if a specific class is referenced with the path, etc that aligns on it's own depending on length of path, as long as it's 2 spaces between the class and the variable name. Only really the parameters that are specifically data types do the whole alignment thing with varying spaces... The amount of "space" adding would be minimal.

```
Example:
/**
 * Create a new database store.
 *
 * @param  \Illuminate\Database\ConnectionInterface  $connection
 * @param  \Illuminate\Contracts\Encryption\Encrypter  $encrypter
 * @param  string  $table
 * @param  string  $prefix
 * @return void
 */

Example:
/**
 * Increment or decrement an item in the cache.
 *
 * @param  string  $key
 * @param  mixed  $value
 * @param  \Closure  $callback
 * @return void
 */
```